### DISCUSSION TITLE
Tabs within Sets within Bard?

### DISCUSSION THREAD

**Discussion by Sennik** [statamic/cms#7010] [General]
I extensively use Bard fields with pre-configured Sets in my templates to make pages as customizabe as possible.

Despite my effort to keep the Bard field and the Sets as clean as possible there are always buttons, sliders and other fields that are not directly primarily needed in everyday use (because by example: they are used to style the specific set). In these cases only the text and image field should be directly accessible when initially opened, the rest secondary after pressing a button ('Show all fields') or selecting a tab.

I know I can achieve this already by using toggles in combination with conditions (do not forget to set 'allways save' on the hidden fields) but I have to set this on every single field I want to be (optionally) hidden/shown. Tip, use a regular toggle instead of the Revealer toggle if you want to save the 'state' of the visiblility.

It would be very handy if I could do this with sections and all fields in these sections at once or even better, use Tabs within the Sets.

**Comment by edalzell** [comment]
This sounds like a feature request to me Sennik, feel free to open one.

**Reply by Sennik** [reply]
https://github.com/statamic/ideas/issues/911

**Comment by Sennik** [comment]
Hi Jason

Totally different.

They ask for a better way to visualize a long list of sets by grouping them in tabs.

I asked for the option to add tabs to sets as a way to group the (possibly many) fields in a set. Frankly, adding a new fieldtype ‘Tabs’  to fieldtypes (Under ‘Special’) to be used to group other fieldtypes within a set.

This already exists as a feature request.

statamic/ideas#361<https://github.com/statamic/ideas/issues/361>

**Reply by jasonvarga** [reply]
Yep I deleted my comment.
